### ISSUE TITLE
Bug Report: No option for allegiance in Rearguard/Spearhead Meeting Engagements

### ISSUE THREAD

**Issue by plavski** [BSData/warhammer-age-of-sigmar#1462]
Currently Using BS Ver: 2.03.21.166
AOS Ver: 61
Order-Kharadron Overlords v27 (Does not matter about dataset)

When trying to create a Meeting Engagements list, there is no option to add an allegiance to the Spearhead or Rearguard sections, leading to roster validation issues that cannot be resolved.

**Comment by cartag** [comment]
This should have been corrected months ago. Please try deleting your data and redownloading.

**Comment by plavski** [comment]
I have tried it on several different devices with a fresh data set each time and the problem persists. There is no option to assign an allegiance for Rearguard or Spearhead forces.

**Comment by cartag** [comment]
Let me rephrase that. The error that caused the validation error has been corrected, so as long as you're loading all parts of the Meeting Engagement in one roster, you won't have the error any more. If you're building each as a separate list, you're going to have the validation errors.

**Comment by plavski** [comment]
![image](https://user-images.githubusercontent.com/15157275/97784877-97767080-1b99-11eb-8b16-991b566794fc.png)

So what am I doing wrong here? Fresh data set, fresh list, Main Body, Rearguard, Spearhead as separate forces within the same roster. Validation errors and no way to add an allegiance in the Rearguard or Spearhead.

**Comment by cartag** [comment]
Sonofabitch, it unfixed itself. Reuploading the fix.

**Comment by cartag** [comment]
I was wrong, I apologize. It's uploaded now and will be in the next release.

**Comment by cartag** [comment]
Pic for proof. 
![image](https://user-images.githubusercontent.com/4388344/97786158-5701fd00-1b6f-11eb-9ed1-65cf1f6ddf05.png)

**Comment by plavski** [comment]
Is this just fixed in KO? I see the same issue in Sylvaneth, for example

![image](https://user-images.githubusercontent.com/15157275/97786232-3737fc80-1ba2-11eb-95bb-a2875e64f4d5.png)

**Comment by cartag** [comment]
Great, looks like I get to tear through everything now...

**Comment by cartag** [comment]
I just went through every modern (non-legacy) catalogue and made sure they're all set up properly. Pushing another release.

**Comment by plavski** [comment]
Cool, thanks for the work. I made a comment on the PR regarding the LoN update as both entries have a `scope` of `roster` which didn't tally with the other updates. This could be me not grokking the full update tho.
